portality/templates/doaj/contact.html

```diff
@@ -45,8 +45,16 @@
     {{ render_field_horizontal(form.message, style="width: 100%; max-width: 400px; min-height: 200px", placeholder="Enter your message here") }}
 
     <div class="form-group">
-        <div class="col-md-9 col-md-offset-3">
-            <button class="btn btn-success" id="send">Send</button>
+        <div class="submit-with-recaptcha col-md-9 col-md-offset-2">
+            <form method="POST">
```

Comment:
This is nesting a form inside a form, which isn't syntactically valid HTML.  Chrome just converts this to a div, so I'd probably do the same here.